packages/orchestrator-ui-components/src/rtk/slices/orchestratorComponentOverride.ts

```diff
@@ -0,0 +1,23 @@
+import { Slice, createSlice } from '@reduxjs/toolkit';
+
+import { ValueOverrideFunction } from '@/components/WfoSubscription/overrides/useSubscriptionDetailValueOverride';
```

Comment:
I would move this type here instead and import it /useSubscriptionDetailValueOverride